README.md

```diff
@@ -19,6 +19,26 @@ Run the following to get started locally:
 
 **Note:** Reach out to your team lead for directions on how to receive the `CHECK_ACCESS_TOKEN`
 
+#### Working with CommonsUI Locally
+
+[CommonsUI](https://github.com/CodeForAfrica/COMMONS-UI) is a library developed and maintained by CfA, that packages React components used to develop various UIs.
+
+To install the commons-ui submodule locally, run:
+
+```sh
+  yarn submodules
+  npm run commons-setup
```

Comment:
If we're using `yarn`, lets use `yarn` everywhere ... no need to mix it up with `npm`